### ISSUE TITLE
[examples] shaders_julia_set doesn't fill screen on Macbook Retina

### ISSUE THREAD

**Issue by a3f** [raysan5/raylib#837]
<img width="912" alt="Screen Shot 2019-05-15 at 18 01 08" src="https://user-images.githubusercontent.com/5204368/57790752-aa924800-773b-11e9-97f7-0405b8dba31e.png">

I am running raylib master with macOS 10.14.4 on a macbook pro retina.

**Comment by raysan5** [comment]
@a3f is high-dpi support flag enabled?

**Comment by a3f** [comment]
`SUPPORT_HIGH_DPI` is defined by default and if I manually undef it, the texture moves:
<img width="912" alt="Screen Shot 2019-05-15 at 23 44 03" src="https://user-images.githubusercontent.com/5204368/57812106-5c496d00-776c-11e9-84a0-a46ca112dd8e.png">

In both cases, the console output is the same:

```
INFO: Display size: 1680 x 1050
INFO: Render size: 800 x 450
INFO: Screen size: 800 x 450
INFO: Viewport offsets: 0, 0
```

**Comment by raysan5** [comment]
@a3f, does it only happen with this example?

**Comment by raysan5** [comment]
Replaced `gl_fragCoord` coordinates by provided `fragTexCoord` (that refer to the full screen texture drawing) on commit https://github.com/raysan5/raylib/commit/9fd410b8a87ede4b40be60ec5043090e2a86c6fe

**Comment by a3f** [comment]
> @a3f, does it only happen with this example?

Just ran them and it happens as well with:

* `shaders_custom_uniform`, `shaders_eratosthenes`, `shaders_postprocessing`

The others are unaffected.

> Replaced gl_fragCoord coordinates by provided fragTexCoord (that refer to the full screen texture drawing) on commit 9fd410b

Still looks the same:
<img width="912" alt="Screen Shot 2019-05-17 at 20 38 34" src="https://user-images.githubusercontent.com/5204368/57949130-d18b7e00-78e3-11e9-8bd4-7a3151907384.png">

**Comment by raysan5** [comment]
ok, I detected the issue, the screenWidth-screenHeight provided to those shaders do not match render size (scaled to HighDPI)

**Comment by MikeDX** [comment]
As an additional feature to this bug, the following code causes the text to be zoomed 2x

```c
BeginTextureMode(F_renderTarget);

    ClearBackground(colour);

    BeginMode3D(camera);
    DrawModel(F_objects[0].model, Vector3Zero(), 1.0f, WHITE);
    DrawGrid(10, 1.0f);         // Draw a grid
    EndMode3D();
  
    DrawTextEx(F_font, text, (Vector2){cur_x,cur_y},fontSize, 0, F_Ink);
    EndTextureMode();
    BeginDrawing();
    ClearBackground(BLACK);

    DrawTextureRec(F_framebuffer.texture, (Rectangle){ 0, 0, F_framebuffer.texture.width, -F_framebuffer.texture.height },
            (Vector2){ 0,0}, WHITE);

    EndDrawing();
```
<img width="1392" alt="Screenshot 2019-12-22 at 23 14 26" src="https://user-images.githubusercontent.com/853931/71328753-20e9fa80-2514-11ea-9096-59322d85103e.png">

however, with the 3d block removed, the text is the correct size
```
BeginTextureMode(F_renderTarget);

    ClearBackground(colour);


    DrawTextEx(F_font, text, (Vector2){cur_x,cur_y},fontSize, 0, F_Ink);
    EndTextureMode();
    BeginDrawing();
    ClearBackground(BLACK);

    DrawTextureRec(F_framebuffer.texture, (Rectangle){ 0, 0, F_framebuffer.texture.width, -F_framebuffer.texture.height },
            (Vector2){ 0,0}, WHITE);

    EndDrawing();
```
<img width="1392" alt="Screenshot 2019-12-22 at 23 14 13" src="https://user-images.githubusercontent.com/853931/71328749-09ab0d00-2514-11ea-98d3-8b8d8383e4ea.png">

**Comment by raysan5** [comment]
`funding needed`: I don't have a Macbook Retina device to work on this issue, if someone could provide one it would be great.

**Comment by raysan5** [comment]
Closing this issue, moved to a more specific description.

**Comment by thevojacek** [comment]
Will this ever get fixed? I am using Raylib 3.0.0 and this is exact behavior i get. What helps is to manually move the window, then everything gets rendered correctly.

**Comment by raysan5** [comment]
@thevojacek actually this issue is related to https://github.com/raysan5/raylib/issues/1086. It's also macOS dependant. No solution found yet...
